changelog/fragments/arbitraryArgs.yaml

```diff
@@ -0,0 +1,8 @@
+entries:
+  - description: >
+      Added the "--ansible-args" command-line flag that allows users to specify arbitrary
+      CLI arguments for ansible-based operators that are passed through ansible-runner.
```

Comment:
```suggestion
      **(Valid just for Ansible-based Operators)** Add new flag `--ansible-args` which allows to specify arbitrarily arguments for ansible-runner.
```

Reply:
I don't think this is only for Ansible-Vault.

Reply:
I'd still call out that we now support ansible vault (as that's the issue that prompted this change), but more as an example rather than as the main thing we added. Also, these are arbitrary arguments for `ansible` not `ansible-runner`

Reply:
So, teh suggestion would be: 

```
 **(Valid just for Ansible-based Operators)** Add new flag `--ansible-args` which allows to specify arbitrarily arguments for Ansible.
```